bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/openshift/windows-machine-config-operator
     support: Red Hat
-  name: windows-machine-config-operator.v7.0.1
+  name: windows-machine-config-operator.v7.1.0
```

Comment:
Shouldn't the skip range be removed for this version?

Reply:
I thought we want to keep 7.0.0 in the skip range so that OLM will update away from it if automatic upgrades are enabled? That's my understanding from [these docs](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.4/html/operators/understanding-the-operator-lifecycle-manager-olm#olm-upgrades-replacing-multiple_olm-understanding-olm) but I could be wrong

```
When searching catalogs for updates, if the head of a channel has a skipRange 
annotation and the currently installed Operator has a version field that falls
in the range, OLM updates to the latest entry in the channel.
```

Reply:
> we want to keep 7.0.0 in the skip range so that OLM will update away from it

Yes to 7.0.1, but for 7.1.0 the upgrade path should go from 7.0.1 -> 7.1.0.